pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -79,6 +79,8 @@ message DNSOption {
     repeated string servers = 1;
     // List of DNS search domains of the cluster.
     repeated string searches = 2;
+    // List of DNS opt strings
+    repeated string opts = 3;
```

Comment:
Sounds good.
This will break the API though, current runtime needs to update the grpc proto after this change, so /cc @feiskyer @mrunalp for heads up.